source/dub/commandline.d

```diff
@@ -278,6 +279,7 @@ struct CommonOptions {
 		args.getopt("bare", &bare, ["Read only packages contained in the current directory"]);
 		args.getopt("v|verbose", &verbose, ["Print diagnostic output"]);
 		args.getopt("vverbose", &vverbose, ["Print debug output"]);
+		args.getopt("verror", &verror, ["Only print errors"]);
 		args.getopt("q|quiet", &quiet, ["Only print warnings and errors"]);
 		args.getopt("vquiet", &vquiet, ["Print no messages"]);
```

Comment:
Nit: I think `verror` should be between `q|quiet` and `vquiet`, so the list is sorted in decreasing level of chattiness.